internal/operator-controller/applier/helm.go

```diff
@@ -109,7 +109,9 @@ func (h *Helm) runPreAuthorizationChecks(ctx context.Context, ext *ocv1.ClusterE
 		preAuthErrors = append(preAuthErrors, fmt.Errorf("authorization evaluation error: %w", authErr))
 	}
 	if len(preAuthErrors) > 0 {
-		return fmt.Errorf("pre-authorization failed: %v", errors.Join(preAuthErrors...))
+		// End the error in a newline since this is a heavily formatted collection of errors such as missing RBAC rules
+		// and we don't want it to run along into any following errors, i.e. the status error wrapping in clusterextension_controller
```

Comment:
Is there an example of where this actually happens? If so, that's probably because we are not using `errors.Join` in _that_ location?

(i.e. if we used errors.Join all the way up the stack, we'd get newlines between all the errors, but no extraneous newlines otherwise)